BGL/include/CGAL/boost/parameter.h

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) 2014  GeometryFactory (France).  All rights reserved.
+//
+// This file is part of CGAL (www.cgal.org); you can redistribute it and/or
+// modify it under the terms of the GNU Lesser General Public License as
+// published by the Free Software Foundation; either version 3 of the License,
+// or (at your option) any later version.
+//
+// Licensees holding a valid commercial license may use this file in
+// accordance with the commercial license agreement provided with the software.
+//
+// This file is provided AS IS with NO WARRANTY OF ANY KIND, INCLUDING THE
+// WARRANTY OF DESIGN, MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE.
+//
+// $URL$
+// $Id$
+// SPDX-License-Identifier: LGPL-3.0+
+// 
+//
+// Author(s)     : Andreas Fabri
+
+#ifndef CGAL_BOOST_PARAMETER_H
+#define CGAL_BOOST_PARAMETER_H
+
+#ifdef BOOST_PARAMETER_MAX_ARITY
+#  if (BOOST_PARAMETER_MAX_ARITY < 12)
+#    error "BOOST_PARAMETER_MAX_ARITY must be at least 12 for CGAL::Mesh_3"
```

Comment:
That looks a little weird as this file might be included by Mesh_2 and if the arity is defined to 8 manually ahead of this file (which is done in the mesh_2 plugin for example), it shouldn't complain that Mesh_3 needs more.

Reply:
Shall I write "at least 12 for CGAL" ?